crates/main/src/mounted_binary.rs

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use fluence_sdk_macro::fce;
+
+use serde::Serialize;
+use serde::Deserialize;
+
+pub const SUCCESS_CODE: i32 = 0;
+
+/// Describes result of calling a CLI service.
+#[fce]
+#[derive(Clone, PartialEq, Default, Eq, Debug, Serialize, Deserialize)]
+pub struct Result {
```

Comment:
Let's add something like
```
impl Result {
   fn into_std(self) -> Option<std::result::Result<String,String>>
}
```

that will check `ret_code` against `BINARY_SUCCESS_CODE` and return Ok or Err. 

P.S. maybe not `std::result::Result`, but a custom Result that is serialization friendly

Reply:
Why Option? I think it's better to make it `unsafe`.

Reply:
Because it could be an invalid string

Reply:
Then it should be `Result<Option<String>, Option<String>>`

Reply:
Dicussed it and came to the idea of having two functions:
`pub fn into_std(self) -> Option<std::result::Result<String, String>>`
`pub fn as_std(&self) -> Option<std::result::Result<String, String>>`